admin_panel/notifications/models.py

```diff
@@ -0,0 +1,95 @@
+"""Models for notifications admin panel app."""
+import uuid
+
+from django.db import models
+from tinymce.models import HTMLField
+
+
+class UUIDMixin(models.Model):
+    """Mixin for id field."""
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+    class Meta:  # noqa: WPS306
+        """Meta."""
+
+        abstract = True
+
+
+class Chanel(models.TextChoices):
```

Comment:
Еще одна опечатка (и дальше еще в таблице, миграциях)
```suggestion
class Channel(models.TextChoices):
```

Reply:
исправлено